WordPress/gutenberg
# Try using the excerpt block to display format like posts

## What?


Idea for #28564

There was plenty of feedback in #53049 about the usefulness of post formats and how although supported they're hard to figure out when one uses a block theme.

## Why?


One thing about post formats is that they overlap a lot with CPTs. To render a special template for a special kind of post we have CPTs which can have their own single and archive templates. Post formats on the other hand use the standard post template but as taxonomy allowed special rendering in the loop, particularly for title less situations (e.g. a status post).

This exploration aims to see if simply fixing the part around query loop rendering is enough to replace the need for post formats: after all a post with one image block is an image post format, a post with one paragraph block can be a status format or a link format etc.

## How?


Makes the excerpt block 

## Testing Instructions





In a block theme:

- open a template , the index or front page are best
- add or edit the query loop in the page by adding in the post template an excerpt block
- from the inspector of the excerpt block toggle on "Behave as title replacement" (TBD)
- add some title less posts with links, videos, images, text
- see the front end and how the loop looks now

### Testing Instructions for Keyboard


N/A

## Screenshots or screencast 



https://github.com/WordPress/gutenberg/assets/107534/d1d74a26-3945-409e-ac05-86d7506c3bd2
